ports/espressif/common-hal/hashlib/Hash.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
```

Comment:
keeping in mind that we'd want to move this out when other mcu families work with the wifi workflow...